README.md

```diff
@@ -0,0 +1,56 @@
+# batwatch
+
+_batwatch_ monitors a system's batteries and UPS units for state changes and sends
+notifications via _gpgmailer_.
+
+Bug fixes are welcome!
+
+## System Power States
+
+_batwatch_ looks at the number of batteries installed on a system and each of those
+batteries' charge levels to determine the system's overall state to be one of the following.
+
+* Fully Charged: All batteries are fully charged and not discharging.
+* Charging: All batteries are either fully charged or charging.
```

Comment:
The description of charging isn't quite accurate. Say instead, at least one battery is charging and no batteries are discharging.